client/src/AddFriend.js

```diff
@@ -76,6 +76,21 @@ class AddFriend extends React.Component {
 
   resetForm () { this.setState({ friendFName: '', friendLName: '', friendEmail: '' }) }
 
+  handleRemove (id) {
+    const newList = this.state.friendList.filter((item) => item.friendId !== id)
+    this.setState({ friendList: newList })
+    axios.put('/api/friendships', ([this.state.userId, id]))
```

Comment:
Apparently `axios.delete` does allow you to send data, it just has a different syntax:
https://stackoverflow.com/questions/51069552/axios-delete-request-with-body-and-headers

We should switch to delete